libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFLoginViewController.m

```diff
@@ -257,18 +270,65 @@ - (UIInterfaceOrientationMask)supportedInterfaceOrientations {
 
 - (void)layoutWebView {
     if (nil != _oauthView) {
+        SFBiometricAuthenticationManagerInternal *bioAuthManager = [SFBiometricAuthenticationManagerInternal shared];
+        BOOL showBioAuthButton = [bioAuthManager locked] && [bioAuthManager showNativeLoginButton];
+        CGFloat heightOffset = showBioAuthButton ? 0.9 : 1.0;
+        
         [_oauthView removeFromSuperview];
         CGFloat x = 0;
         CGFloat y = [self belowFrame:self.navBar.frame];
         CGFloat w = self.view.bounds.size.width;
-        CGFloat h = self.view.bounds.size.height - y;
+        CGFloat h = (self.view.bounds.size.height - y) * heightOffset;
         self.oauthView.frame = CGRectMake(x, y, w, h);
         [self.view addSubview:_oauthView];
+        
+        if (showBioAuthButton) {
+            UIButton *button = [UIButton buttonWithType:UIButtonTypeCustom];
+            [button setTitle:[SFSDKResourceUtils localizedString:@"biometricLoginButton"] forState:UIControlStateNormal];
+            x = 0;
+            y = (self.view.bounds.size.height - y);
+            w = self.view.bounds.size.width;
+            h = ((self.view.bounds.size.height - y) * 0.1);
+            button.frame = CGRectMake(x, y, w, h);
+            [button addTarget:self action:@selector(presentBioAuthAction:) forControlEvents:UIControlEventTouchUpInside];
+            [self.view addSubview:button];
+        }
```

Comment:
There is probably a better way to do this.

Reply:
@bbirman do we have a "Login with IDP" button shown on the Login screen on iOS? If there is, how are we handling the case where "Login with IDP" and "Biometric Login" buttons both need to be shown?

Reply:
This is how it looks with IDP, so it might work out that if you select "Login Using the App", then you'd get the "Biometric Login" button on the regular login screen?
<img width="564" alt="Screenshot 2023-05-09 at 9 16 49 PM" src="https://github.com/forcedotcom/SalesforceMobileSDK-iOS/assets/840431/4e32fc18-31d1-45d6-abcf-f10bca1a2d37">

Reply:
Yeah, I figured that screen had not changed.  It would be nice to have more consistent UX between Android and iOS.